client/src/document/forms/create.scss

```
@@ -0,0 +1,6 @@
+.document-create {
+  .close {
+    font-size: 70%;
+    margin-left: 20px;
+  }
```

We have variables in minimalist that we should stick to, for this, I would recommend

```suggestion
  .close {
    font-size: $tiny-text;
    margin-left: $base-spacing;
  }
```